workflows-scripts/item-search-filter/InventoryItemSearch.jmx

```diff
@@ -0,0 +1,1774 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.2.1">
+  <hashTree>
```

Comment:
How about using `natureOfContentTermIds `which aren't connected to bugfest environment?

`GET instance-storage/instances?query=(languages=spa and natureOfContentTermIds=ebbbdef1-00e1-428b-bc11-314dc0705074) sortby title`